packages/mobile/src/fiatExchanges/ProviderOptionsScreen.tsx

```diff
@@ -127,42 +128,58 @@ function ProviderOptionsScreen({ route, navigation }: Props) {
     SIMPLEX_RESTRICTED,
     RAMP_RESTRICTED,
     TRANSAK_RESTRICTED,
+    XANPOOL_RESTRICTED,
   } = getProviderAvailability(userLocation)
 
+  const xanpool = {
+    id: CicoProviderNames.Xanpool,
+    paymentMethods: [PaymentMethod.Card, PaymentMethod.Bank],
+    restricted: XANPOOL_RESTRICTED,
+    onSelected: () =>
+      navigate(Screens.XanpoolScreen, {
+        localAmount: route.params.amount,
+        currencyCode: localCurrency,
```

Comment:
Should we add `|| FALLBACK_CURRENCY` like the other providers do?

Reply:
We actually don't need this because we already have that logic built in to the `getLocalCurrencyCode` selector. I deleted it from the other providers in the Simplex PR.